tools/build/ms-plugins.coffee

```diff
@@ -0,0 +1,113 @@
+{ walkFiles } = require('@resin.io/doxx-utils')
+HbHelper = require('@resin.io/doxx-handlebars-helper')
+
+path = require('path')
+fse = require('fs-extra')
+semver = require('semver')
+
+utils = require('../lib/utils')
+pythonHelper = require('../lib/python')
+
+
+removeFile = (obj, file, files) ->
+	if obj and obj is utils.UNSUPPORTED
+		delete files[file]
+
+copyFile = (src, dest) ->
+	try
+		fse.copySync(src, dest)
+	catch err
```

Comment:
yeah, I don't want an error can break the whole build, just log and we can fix later.